src/guides/v2.3/graphql/develop/extend-existing-schema.md

```diff
@@ -0,0 +1,105 @@
+---
+group: graphql
+title: Extend an existing GraphQL schema
+contributor_name: Adarsh Manickam
+contributor_link: https://github.com/drpayyne
+---
+
+The Magento GraphQL system is built in a way so that you can extend existing GraphQL schemas to add more fields, and modify existing resolver behavior, among other extension points. This works similar to the XML merging mechanism present. All `schema.graphqls` files are merged together to a single schema file. In this process, all nodes with the same type (type, interface, enum, etc.) and name are merged together and recursively extended/overriden, just like how XML merging works.
+
+## Extend schema
+
+The first step to retrieve a custom field along with an existing query is to extend the appropriate schema object.
+
+To add a new/custom field to the database, please refer to the [Declarative Schema]({{ page.baseurl }}/extension-dev-guide/declarative-schema.html) documentation for more information on that.
+
+In our example, we aren't going to add a new field to the database, but rather, change the description of an existing field (`attribute_set_id`), and add a new field (`attribute_set_name`) to the GraphQL schema for the products query.
+
+The simplified structure of the query schema to get products is:
+
+```
```

Comment:
What language do I specify for the code block? (since the linting test is failing)